vmdb/app/models/mixins/naming_index_mixin.rb

```diff
@@ -0,0 +1,20 @@
+module NamingIndexMixin
```

Comment:
I've been thinking about the name of this, and I don't like the term index.  It's way too conflated, since there are indexes in SQL, and index in an Array, etc.  Plus from a reader's perspective, it's not clear that a NamingIndex is really a sequence.  I think a better name might be NamingSequenceMixin, with has_many :naming_sequences.